README.md

```diff
@@ -1,81 +1,76 @@
-```
-      :::::::::  ::::::::::     :::      :::::::: ::::::::::: ::::::::::: ::::::::  ::::    :::
-     :+:    :+: :+:          :+: :+:   :+:    :+:    :+:         :+:    :+:    :+: :+:+:   :+:
-    +:+    +:+ +:+         +:+   +:+  +:+           +:+         +:+    +:+    +:+ :+:+:+  +:+
-   +#++:++#:  +#++:++#   +#++:++#++: +#+           +#+         +#+    +#+    +:+ +#+ +:+ +#+
-  +#+    +#+ +#+        +#+     +#+ +#+           +#+         +#+    +#+    +#+ +#+  +#+#+#
- #+#    #+# #+#        #+#     #+# #+#    #+#    #+#         #+#    #+#    #+# #+#   #+#+#
-###    ### ########## ###     ###  ########     ###     ########### ########  ###    ####
-      :::::::::: ::::::::  :::::::::   ::::::::  ::::::::::
-     :+:       :+:    :+: :+:    :+: :+:    :+: :+:
-    +:+       +:+    +:+ +:+    +:+ +:+        +:+
-   :#::+::#  +#+    +:+ +#++:++#:  +#+        +#++:++#
-  +#+       +#+    +#+ +#+    +#+ +#+        +#+
- #+#       #+#    #+# #+#    #+# #+#    #+# #+#
-###        ########  ###    ###  ########  ##########
-```
+          :::::::::  ::::::::::     :::      :::::::: ::::::::::: ::::::::::: ::::::::  ::::    :::
+         :+:    :+: :+:          :+: :+:   :+:    :+:    :+:         :+:    :+:    :+: :+:+:   :+:
+        +:+    +:+ +:+         +:+   +:+  +:+           +:+         +:+    +:+    +:+ :+:+:+  +:+
+       +#++:++#:  +#++:++#   +#++:++#++: +#+           +#+         +#+    +#+    +:+ +#+ +:+ +#+
+      +#+    +#+ +#+        +#+     +#+ +#+           +#+         +#+    +#+    +#+ +#+  +#+#+#
+     #+#    #+# #+#        #+#     #+# #+#    #+#    #+#         #+#    #+#    #+# #+#   #+#+#
+    ###    ### ########## ###     ###  ########     ###     ########### ########  ###    ####
+          :::::::::: ::::::::  :::::::::   ::::::::  ::::::::::
+         :+:       :+:    :+: :+:    :+: :+:    :+: :+:
+        +:+       +:+    +:+ +:+    +:+ +:+        +:+
+       :#::+::#  +#+    +:+ +#++:++#:  +#+        +#++:++#
+      +#+       +#+    +#+ +#+    +#+ +#+        +#+
+     #+#       #+#    #+# #+#    #+# #+#    #+# #+#
+    ###        ########  ###    ###  ########  ##########
 
 ## Meta
 
-* __State:__ development
-* __Demo:__ https://artsy.github.io/reaction-force
-* __CI:__ [![CircleCI](https://circleci.com/gh/artsy/reaction-force.svg?style=shield)](https://circleci.com/gh/artsy/reaction-force)
-* __Point People:__ [@alloy](https://github.com/alloy) & [@broskoski](https://github.com/broskoski)
+-   **State:** development
+-   **Demo:** <https://artsy.github.io/reaction>
+-   **CI:** [![CircleCI](https://circleci.com/gh/artsy/reaction.svg?style=shield)](https://circleci.com/gh/artsy/reaction)
+-   **Point People:** [@alloy](https://github.com/alloy) & [@l2succes](https://github.com/l2succes)
```

Comment:
I’m not a fan of these indents, I find the 1 space version to be more clear about what the different items are.